x-pack/functionbeat/magefile.go

```diff
@@ -9,29 +9,59 @@ package main
 import (
 	"context"
 	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
 	"time"
 
 	"github.com/magefile/mage/mg"
-	"github.com/magefile/mage/sh"
 
```

Comment:
It would be nice if you could utilize as many of these common targets as possible:

```
	// mage:import
	_ "github.com/elastic/beats/dev-tools/mage/target/common"
	// mage:import
	//_ "github.com/elastic/beats/dev-tools/mage/target/build" # You can use this one because of the custom Build() target.
	// mage:import
	_ "github.com/elastic/beats/dev-tools/mage/target/pkg"
	// mage:import
	_ "github.com/elastic/beats/dev-tools/mage/target/dashboards"
	// mage:import
	_ "github.com/elastic/beats/dev-tools/mage/target/test"
	// mage:import
	"github.com/elastic/beats/dev-tools/mage/target/unittest"
	// mage:import
	"github.com/elastic/beats/dev-tools/mage/target/integtest"
```

Reply:
How can I use `target/build`? I am not sure I understand your comment.